src/Auth/EmailVerification/EmailVerificationRepository.php

```diff
@@ -0,0 +1,206 @@
+            ->first();
+    }
+
+    /**
+     * Generate and send a email with link instructions
+     *
+     * @author Paulius Navickas <pana@nodes.dk>
+     *
+     * @access public
+     * @param  array $conditions WHERE conditions to locate user. Format: ['column' => 'value']
+     * @return boolean
+     * @throws \Nodes\Api\Auth\Exceptions\ResetPasswordNoUserException
+     */
+    public function sendVerificationEmail($user)
+    {
+        if (empty($user)) {
+            throw new ResetPasswordNoUserException('Could not find any user with those credentials');
+        }
+
+        // Generate email verification token
+        $token = $this->generateEmailVerificationToken($user);
+
+        // Generate email verification domain
+        $domain = env('NODES_ENV', false) ? sprintf('https://%s.%s', env('APP_NAME'), env('APP_DOMAIN')) : config('app.url');
+
+        // Send e-mail with instructions on how to reset password
+        $status = (bool) Mail::send([
+            'html' => config('nodes.api.email-verification.views.html', 'nodes.api::email-verification.emails.html'),
+            'text' => config('nodes.api.email-verification.views.text', 'nodes.api::email-verification.emails.text')
+        ], [
+            'user' => $user,
+            'domain' => $domain,
+            'email' => $user->email,
+            'token' => $token,
+            'expire' => config('nodes.api.email-verification.expire'),
+            'senderName' => (config('nodes.api.email-verification.from.name') != 'Nodes') ? config('nodes.api.email-verification.from.name') : config('nodes.project.name')
+        ], function($message) use ($user) {
+            $message->to($user->email)
+                ->from(config('nodes.api.remail-verification.from.email', 'no-reply@nodes.dk'), config('nodes.api.email-verification.from.name', 'Nodes'))
+                ->subject(config('nodes.api.email-verification.subject', 'Email verification request'));
+        });
+
+        return $status;
+    }
+
+    /**
+     * Generate reset token
+     *
+     * @author Morten Rugaard <moru@nodes.dk>
+     *
+     * @access protected
+     * @param  \Illuminate\Database\Eloquent\Model $user
+     * @return string
+     */
+    protected function generateEmailVerificationToken(IlluminateModel $user)
+    {
+        // Generate new token using Laravel's encryption key
+        $token = hash_hmac('sha256', str_random(40), config('app.key'));
+
+        // Expire timestamp
+        $expire = Carbon::now()->addMinutes(config('nodes.api.reset-password.expire', 120));
+
+        // If user has previously tried to reset his/her password
+        // we should just update the token of the previous entry
+        // instead of creating a new one
+        $verificationToken = $this->where('email', '=', $user->email)->first();
+        if (!empty($verificationToken)) {
+            $verificationToken->update(['token' => $token, 'used' => 0, 'expire_at' => $expire->format('Y-m-d H:i:s')]);
```

Comment:
Using incorrect field since you've renamed the field from `token` to `verification`